### ISSUE TITLE
Does remark-latest.min.js still work?

### ISSUE THREAD

**Issue by rdhutchins** [gnab/remark#225]
Examples on https://github.com/gnab/remark say to include remark like this:

```
    <script src="https://gnab.github.io/remark/downloads/remark-latest.min.js">
    </script>
    <script>
      var slideshow = remark.create();
    </script>
```

But the latest download package at https://github.com/gnab/remark/downloads is from Dec 2012, and it seems there have been edits since then.

Is the one on gnab.github.io different?

**Comment by gnab** [comment]
GitHub discontinued the downloads section a while back, so everything under https://github.com/remark/downloads (without `gnab` in front) is no longer maintained. I am not in control of all the examples, and they should work fine with the old version.

To get the latest changes, however, https://gnab.github.io/remark/downloads/remark-latest.min.js which is updated on every deplay should be used. This is also the ones referenced in the README and the wiki.

**Comment by rdhutchins** [comment]
Got it, thanks.  How can I find out what version is on latest?

**Comment by gnab** [comment]
In the next release, `remark.version`, will be available. Just leave this issue open, and I'll close it once version 0.11 is relased.

**Comment by gnab** [comment]
Version 0.11 has now been released, fixing this and #200.
